CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+* Added `conn.Ping(ctx)` method for check availability of `conn.Conn`
+* Refactored `cluster.Cluster.Get(ctx)` for return only available connection (instead return any connection from balancer)
+* Added address to error description which thrown from `conn.take()`
+* Renamed package `internal/db` to `internal/database` for exclude collisions with variable name `db`
```

Comment:
Renamed package `internal/db` to `internal/database` **to** exclude collisions with variable name `db`